src/main/java/transforms/CombineDateTimeTransform.java

```diff
@@ -30,32 +30,30 @@
      * @param datePattern: list of patterns of date
      * @param timeColumn:  column of time
      * @param timePattern: list of patterns of time
-     * @param timezone: name of the timezone
+     * @param timezone:    name of the timezone
      */
     @JsonCreator
     public CombineDateTimeTransform(
             @JsonProperty( Constants.DATE_COLUMN ) String dateColumn,
             @JsonProperty( Constants.DATE_PATTERN ) String[] datePattern,
             @JsonProperty( Constants.TIME_COLUMN ) String timeColumn,
             @JsonProperty( Constants.TIME_PATTERN ) String[] timePattern,
-            @JsonProperty( Constants.TIMEZONE ) Optional<String> timezone
+            @JsonProperty( Constants.TIMEZONE ) String timezone
```

Comment:
have you tested actually running an integration against this without passing a timezone?

Reply:
yeah, it nullifies the timezone.